### ISSUE TITLE
[10.7.0-rc3] Metadata editor for TV shows fails to load

### ISSUE THREAD

**Issue by joshuaboniface** [jellyfin/jellyfin#5202]
**Describe the bug**
When using the Metadata editor, expanding a TV Show fails and throws an error. The expansion loads indefinitely.

**System (please complete the following information):**
 - OS: Debian
 - Virtualization: KVM VM
 - Clients: Browser
 - Browser: Firefox 78.6.1esr (64-bit)
 - Jellyfin Version: 10.7.0-rc3
 - Playback: N/A
 - Installed Plugins: N/A
 - Reverse Proxy: N/A
 - Base URL: N/A
 - Networking: N/A
 - Storage: N/A

**To Reproduce**
1. Go to 'Metadata Editor'
2. Expand the 'TV Shows' category
3. Expand a show's group

**Expected behavior**
The show's seasons should be shown.

**Logs**
```
Feb 10 00:38:22 jf1 jellyfin[481]: [00:38:22] [ERR] [41] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request. URL GET /Users/b5f6371d-9b82-4264-bae2-f623d0c223bf/Items.                                    
Feb 10 00:38:22 jf1 jellyfin[481]: System.NullReferenceException: Object reference not set to an instance of an object.                                                                                                          
Feb 10 00:38:22 jf1 jellyfin[481]:    at Emby.Server.Implementations.Dto.DtoService.AttachBasicFields(BaseItemDto dto, BaseItem item, BaseItem owner, DtoOptions options)                                                        
Feb 10 00:38:22 jf1 jellyfin[481]:    at Emby.Server.Implementations.Dto.DtoService.GetBaseItemDtoInternal(BaseItem item, DtoOptions options, User user, BaseItem owner)                                                         
Feb 10 00:38:22 jf1 jellyfin[481]:    at Emby.Server.Implementations.Dto.DtoService.GetBaseItemDtos(IReadOnlyList`1 items, DtoOptions options, User user, BaseItem owner)                                                        
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Api.Controllers.ItemsController.GetItems(Nullable`1 userId, String maxOfficialRating, Nullable`1 hasThemeSong, Nullable`1 hasThemeVideo, Nullable`1 hasSubtitles, Nullable`1 ha
sSpecialFeature, Nullable`1 hasTrailer, String adjacentTo, Nullable`1 parentIndexNumber, Nullable`1 hasParentalRating, Nullable`1 isHd, Nullable`1 is4K, LocationType[] locationTypes, LocationType[] excludeLocationTypes, Nullable`1 isMissin
g, Nullable`1 isUnaired, Nullable`1 minCommunityRating, Nullable`1 minCriticRating, Nullable`1 minPremiereDate, Nullable`1 minDateLastSaved, Nullable`1 minDateLastSavedForUser, Nullable`1 maxPremiereDate, Nullable`1 hasOverview, Nullable`1
 hasImdbId, Nullable`1 hasTmdbId, Nullable`1 hasTvdbId, Guid[] excludeItemIds, Nullable`1 startIndex, Nullable`1 limit, Nullable`1 recursive, String searchTerm, String sortOrder, Nullable`1 parentId, ItemFields[] fields, String[] excludeIt
emTypes, String[] includeItemTypes, ItemFilter[] filters, Nullable`1 isFavorite, String[] mediaTypes, ImageType[] imageTypes, String sortBy, Nullable`1 isPlayed, String[] genres, String[] officialRatings, String[] tags, Int32[] years, Null
able`1 enableUserData, Nullable`1 imageTypeLimit, ImageType[] enableImageTypes, String person, Guid[] personIds, String[] personTypes, String[] studios, String[] artists, Guid[] excludeArtistIds, Guid[] artistIds, Guid[] albumArtistIds, Gu
id[] contributingArtistIds, String[] albums, Guid[] albumIds, Guid[] ids, VideoType[] videoTypes, String minOfficialRating, Nullable`1 isLocked, Nullable`1 isPlaceHolder, Nullable`1 hasOfficialRating, Nullable`1 collapseBoxSetItems, Nullab
le`1 minWidth, Nullable`1 minHeight, Nullable`1 maxWidth, Nullable`1 maxHeight, Nullable`1 is3D, SeriesStatus[] seriesStatus, String nameStartsWithOrGreater, String nameStartsWith, String nameLessThan, Guid[] studioIds, Guid[] genreIds, Bo
olean enableTotalRecordCount, Nullable`1 enableImages)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Api.Controllers.ItemsController.GetItemsByUserId(Guid userId, String maxOfficialRating, Nullable`1 hasThemeSong, Nullable`1 hasThemeVideo, Nullable`1 hasSubtitles, Nullable`1
hasSpecialFeature, Nullable`1 hasTrailer, String adjacentTo, Nullable`1 parentIndexNumber, Nullable`1 hasParentalRating, Nullable`1 isHd, Nullable`1 is4K, LocationType[] locationTypes, LocationType[] excludeLocationTypes, Nullable`1 isMiss
ing, Nullable`1 isUnaired, Nullable`1 minCommunityRating, Nullable`1 minCriticRating, Nullable`1 minPremiereDate, Nullable`1 minDateLastSaved, Nullable`1 minDateLastSavedForUser, Nullable`1 maxPremiereDate, Nullable`1 hasOverview, Nullable
`1 hasImdbId, Nullable`1 hasTmdbId, Nullable`1 hasTvdbId, Guid[] excludeItemIds, Nullable`1 startIndex, Nullable`1 limit, Nullable`1 recursive, String searchTerm, String sortOrder, Nullable`1 parentId, ItemFields[] fields, String[] exclude
ItemTypes, String[] includeItemTypes, ItemFilter[] filters, Nullable`1 isFavorite, String[] mediaTypes, ImageType[] imageTypes, String sortBy, Nullable`1 isPlayed, String[] genres, String[] officialRatings, String[] tags, Int32[] years, Nu
llable`1 enableUserData, Nullable`1 imageTypeLimit, ImageType[] enableImageTypes, String person, Guid[] personIds, String[] personTypes, String[] studios, String[] artists, Guid[] excludeArtistIds, Guid[] artistIds, Guid[] albumArtistIds,
Guid[] contributingArtistIds, String[] albums, Guid[] albumIds, Guid[] ids, VideoType[] videoTypes, String minOfficialRating, Nullable`1 isLocked, Nullable`1 isPlaceHolder, Nullable`1 hasOfficialRating, Nullable`1 collapseBoxSetItems, Null
able`1 minWidth, Nullable`1 minHeight, Nullable`1 maxWidth, Nullable`1 maxHeight, Nullable`1 is3D, SeriesStatus[] seriesStatus, String nameStartsWithOrGreater, String nameStartsWith, String nameLessThan, Guid[] studioIds, Guid[] genreIds,
Boolean enableTotalRecordCount, Nullable`1 enableImages)
Feb 10 00:38:22 jf1 jellyfin[481]:    at lambda_method316(Closure , Object , Object[] )
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[]
 arguments)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
Feb 10 00:38:22 jf1 jellyfin[481]: --- End of stack trace from previous location ---
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
Feb 10 00:38:22 jf1 jellyfin[481]: --- End of stack trace from previous location ---
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean
 isCompleted)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
Feb 10 00:38:22 jf1 jellyfin[481]: --- End of stack trace from previous location ---
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.ServerStartupMessageMiddleware.Invoke(HttpContext httpContext, IServerApplicationHost serverApplicationHost, ILocalizationManager localizationManager)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.WebSocketHandlerMiddleware.Invoke(HttpContext httpContext, IWebSocketManager webSocketManager)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.IpBasedAccessValidationMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager, IServerConfigurationManager serverConfigurationManager)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.LanFilteringMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager, IServerConfigurationManager serverConfigurationManager)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationRes
ult authorizeResult)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Swashbuckle.AspNetCore.ReDoc.ReDocMiddleware.Invoke(HttpContext httpContext)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.RobotsRedirectionMiddleware.Invoke(HttpContext httpContext)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.LegacyEmbyRouteRewriteMiddleware.Invoke(HttpContext httpContext)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.Invoke(HttpContext context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.ResponseTimeMiddleware.Invoke(HttpContext context)
Feb 10 00:38:22 jf1 jellyfin[481]:    at Jellyfin.Server.Middleware.ExceptionMiddleware.Invoke(HttpContext context)
```

**Screenshots**
![Screenshot from 2021-02-10 00-55-47](https://user-images.githubusercontent.com/4031396/107471189-bb533600-6b3a-11eb-8564-712048935ea0.png)

**Additional context**
Cannot verify if this existed in 10.6.z or is a regression.

**Comment by joshuaboniface** [comment]
I'm calling this one release critical if only because it completely breaks the metadata editor for TV Shows. If it's just a me problem we can close it, but needs verification.

**Comment by crobibero** [comment]
Possibly fixed by https://github.com/jellyfin/jellyfin/pull/5105

**Comment by GermanCoding** [comment]
This seems to be fixed on the current release-10.7.z branch. Since updating my local builds today I can no longer reproduce, I think a934477850a19fae6f93e82c9dd1cc12f036330d fixed it.

**Comment by undead9786** [comment]
Is there a release date for rc4 with the fix? I keep having to switch between 1.6.4 and 1.7.0 rc3 when doing TV Show addictions.

**Comment by mooninite** [comment]
I'm running RC4 and the issue appears to be fixed to me.
